app/posts/register-trainee-teachers/2020-11-27-beta-prototype-research-round-3.md

```diff
@@ -0,0 +1,169 @@
+
+>“I know the course codes for Primary salaried and Primary training by heart. Otherwise I’d look on UCAS but I don’t have to on Manage. Internally we’d write the subject title.”
+
+>“Useful that this has already been populated - I assume it comes through Publish?”
+
+Subject and course code seemed to be the most important details needed to identify a course. We [adjusted the prototype](/images/register-trainee-teachers/picking-publish-courses/asking-for-publish-course.png) to respond to this feedback during the testing round - removing data that did not help users identify a course. 
+
+The current design works well for picking from around 10 courses or less - but we have providers that offer up to 90 courses, and we need to make sure our solution works for them too.
+
+> “Title and code is enough for me. Differentiating by primary and secondary might help, if you’ve got a lot of codes.”
+
+### Confirming data input and error messages
+
+Users liked the idea of checking their work using the section confirmation pages  and understood the progress labels. 
+
+> "The ‘Completed’ button says ‘I’m happy with this section‘, and that this is ok to be saved against that record."
+
+> "There could be certain sections where I’d want to save a partially complete record. [It should then show] in progress or draft."
+
+Users also understood different error messages and were able to adjust data to submit easily.     
+
+### Negative assessment outcome
+
+Our flow around assessment failure does not reflect the real process.
+
+#### What we thought happens
+
+Trainees go through a standard assessment and can either pass or fail it. If they fail it, they either drop out or get a course extension at the end of which they are being re-assessed.
+
+#### What really happens
+
+Currently a trainee is either withdrawn if they’re unlikely to pass assessment, or they get an extension if they’re likely to pass but need a bit more time.
+
+'Failing' is considered a rare exception. Failing and then staying on the course is entirely unrealistic.
+
+>“We’ve never had anybody fail. Something would be implemented before that, a remediation stage - 3 week period to make sure they pass. Or we extend the course in which case I would change the [programme] date.”
+
+>“Normally they would withdraw before this point. They’d be informed of this process, it has to be ratified by executive boards, lots of paperwork before this point. We wouldn’t offer an extension to someone who isn’t going to get there regardless.”
+
+Read more on what [our users said about 'failing'](/register-trainee-teachers/recording-training-outcomes-iteration-2/#what-our-users-said).
+
+Users act responsibly when it comes to updating outcomes on DTTP, but because an extension is not considered a formal "outcome" it is currently either not recorded at all or falls under deferrals.
```

Comment:
```suggestion
Users act responsibly when it comes to updating outcomes on DTTP, but because an extension is not considered a formal “outcome” it is currently either not recorded at all or falls under deferrals.
```